io/plugins/eu.esdihumboldt.hale.io.topojson/src/eu/esdihumboldt/hale/io/topojson/TopoJsonInstanceWriter.java

```diff
@@ -171,8 +177,60 @@ private void convertShapefileToTopoJson(String shapefilePath,
 			CoordinateReferenceSystem sourceCrs, String targetPath, String topologyName, int kink,
 			int quantizeDigit, boolean compressOutput) throws IOException {
 
-		TopojsonApi.shpToTopojsonFile(shapefilePath, sourceCrs, targetPath, topologyName, kink,
+		String aJson = TopojsonApi.shpToTopojson(shapefilePath, sourceCrs, topologyName, kink,
 				quantizeDigit, compressOutput);
+		try {
+			// Parse JSON string
+			ObjectMapper mapper = new ObjectMapper();
+			JsonNode jsonNode = mapper.readTree(aJson);
+
+			// Replace "\u0000" values with null recursively
+			replaceUnwantedCharacters(jsonNode);
+
+			// Convert modified JSON node back to string
+			String modifiedJsonString = mapper.writeValueAsString(jsonNode);
+
+			if (compressOutput) {
+				Toolbox.writeFile(targetPath, Compress.compressB64(modifiedJsonString));
+			}
+			else {
+				Toolbox.writeFile(targetPath, modifiedJsonString);
+			}
+
+		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
The exception should not be dropped silently but end up in the report and ideally make it fail. You should be able to achieve that by letting this method throw it.